src/main/java/easton/Easton.java

```diff
@@ -0,0 +1,263 @@
+package easton;
```

Comment:
Package naming is in lower case. Class names are in PascalCase and variable names are in camelCase. Great job for following the naming conventions.